src/Symfony/Bundle/FrameworkBundle/Command/SecretsAddCommand.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\DependencyInjection\Secrets\JweHandler;
+use Symfony\Component\DependencyInjection\SecretVarProcessor;
+
+/**
+ * Console command to add an encrypted secret
+ * Usage: php bin/console secrets:add
+ */
+class SecretsAddCommand extends Command
+{
+    protected static $defaultName = 'secrets:add';
+    private $io;
+    private $secretsProcessor;
+
+    public function __construct(SecretVarProcessor $secretsProcessor)
+    {
+        $this->secretsProcessor = $secretsProcessor;
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Encrypt and add a secret key value pair to the specified secrets file')
+            ->setHelp(<<<'HELP'
+The <info>%command.name%</info> command adds an encrypted secret to the specified secrets-file
+
+If encrypted_secrets.enabled is set to true in your yaml config,
```

Comment:
What happens if this is false? Probably it should throw an error that it needs to be enabled?

Reply:
yeah if we're going to remove the runtime configuration options i will update all of this